orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccessTokenService.java

```diff
@@ -1,85 +0,0 @@
-package uk.gov.di.orchestration.shared.services;
```

Comment:
Wanted to double check that this is correct - should orch and auth both have this service. Or does only auth components use this service ?

Reply:
Yeah, it's unused. If you take a look at it in InteliJ, the only reference to any of the contractors of that class is from  within another one of it's contractors. I had to modify Auth's copy, but thought it made more sense to just delete ours.